turnitintooltwo_view.class.php

```diff
@@ -1472,7 +1473,8 @@ public function get_submission_inbox_row($cm, $turnitintooltwoassignment, $parts
         if (!$istutor) {
             $data = array($partid, $checkbox, $studentname, $rawtitle, $title, $objectid, $rawmodified, $modified);
         } else {
-            $data = array($partid, $checkbox, $studentlastname, $studentname, $rawtitle, $title, $objectid, $rawmodified, $modified);
+            $studentfirstname = $submission->firstname;
+            $data = array($partid, $checkbox, $studentlastname, $studentname, $rawtitle, $title, $objectid, $rawmodified, $modified, $studentfirstname);
```

Comment:
@ixiongtii `How come we change this from 3 -> 9?`

we have to add one additional item to first initial table columns array, and shift all others columns, 3 and 9 it is indexes, I didn't replace it because logic really not obvious, and $studentname it is not really string, it is a link tag, and this link opens additional student view, so I decided add additional logic instead of touch existing